data_safe_haven/types/enums.py

```diff
@@ -157,7 +164,15 @@ class PermittedDomains(tuple[str, ...], Enum):
     )
     MICROSOFT_GRAPH_API = ("graph.microsoft.com",)
     MICROSOFT_LOGIN = ("login.microsoftonline.com",)
+    MICROSOFT_CONTAINER_REGISTRY = ("mcr.microsoft.com", "*.data.mcr.microsoft.com")
     MICROSOFT_IDENTITY = MICROSOFT_GRAPH_API + MICROSOFT_LOGIN
+    MANAGED_IDENTITIES = (
+        "*.identity.azure.net",
+        *MICROSOFT_LOGIN,
+        "*.login.microsoftonline.com",
+        "*.login.microsoft.com",
+    )
```

Comment:
```suggestion
    AZURE_MANAGED_IDENTITIES = MICROSOFT_LOGIN + (
        "*.identity.azure.net",
        "*.login.microsoft.com",
    )
```

unless order is important?

Reply:
It's not: I'll work on the renaming and re-ordering. Although Ruff insists in unpacking `MICROSOFT_LOGIN`.